common/core/desktop/src/debug.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Keyman Core - Debug class definition
+ */
+
+#pragma once
+
+#include <cassert>
+#include <vector>
+
+#include <keyman/keyboardprocessor.h>
+#include <keyman/keyboardprocessor_debug.h>
+
+namespace km {
+namespace kbp
+{
+
+using debug_item = km_kbp_state_debug_item;
```

Comment:
Why do we define this alias? I'm not sure it adds clarity.

Reply:
It just matches what we are doing with `action_item`; I agree with you that it is unnecessary. Removing it.